content/en/blog/2023/otel-arrow/index.md

```diff
@@ -0,0 +1,237 @@
+---
+title: Reduce Telemetry Traffic by Up to 2 Times Using OTel Arrow Protocol
```

Comment:
"Reduce" and "up to 2 times" doesn't flow well. How about a shorter title like:

```suggestion
title: Reduce Telemetry Traffic by Half Using the OTel Arrow Protocol
```

Reply:
@jmacd and I ended up choosing a different title.